docs/_static/extra.css

```diff
@@ -0,0 +1,52 @@
+img.align-center {
```

Comment:
Let's provide this via `third_party/open-source-pdks` submodule by adding the file to `_static` under https://github.com/google/open-source-pdks/tree/main/docs

Reply:
This can be done for all open-source-pdks without any problems.